docs/design/hbar-limiter.md

```diff
@@ -178,6 +184,52 @@ classDiagram
     EthAddressHbarSpendingPlanRepository --> CacheService : uses
     IpAddressHbarSpendingPlanRepository --> CacheService : uses
 ```
+### Support flexible alerting mechanisms for spending thresholds
+The initial spending threshold of the Tier 3 General users will be a rough
+estimate based on the current daily spending of the Relay Operator.  In order
+to refine this over time a prometheus metric called the `basicSpendingPlanCounter`
+will be used to track the number of unique spending plans.  
+
+The metrics listed below will be added to help determine the best Tier 3 General users
+over time:
+
+1. Daily Unique Users Counter
+2. Average Daily Users
```

Comment:
basically, #1 is the total of unique users and #2 is the total users for each day, right?

Reply:
Yes.